code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -291,8 +315,8 @@
 			message = "<B>[comm_paygrade][src]</B> yawns."
 
 		if("help")
-			var/msg = {"<br><br><b>To use an emote, type an asterix (*) before a following word. Emotes with a sound are <span style='color: green;'>green</span>. Spamming emotes with sound will likely get you in trouble, don't do it.<br><br> \
-blink, blink_r, bow-(mob name), chuckle, <span style='color: green;'>clap</span>, collapse, cough, cry, drool, eyebrow, facepalm, 
+			var/msg = {"<br><br><b>To use an emote, type an asterix (*) before a following word. Emotes with a sound are <span style='color: green;'>green</span>. Emotes that are <span style='color: red;'>RED</span> are done at your own risk. Spamming emotes with sound will likely get you in trouble, don't do it.<br><br> \
+blink, blink_r, bow-(mob name), chuckle, <span style='color: green;'>clap</span>, collapse, cough, <span style='color: red;'>dab</span>, cry, drool, eyebrow, facepalm, 
```

Comment:
Since these emotes are listed in alphabetic order, **dab** should come after **cry**.